crates/runtime/src/dataconnector/debezium.rs

```diff
@@ -246,13 +250,15 @@ impl DataConnector for Debezium {
                 .boxed()
                 .context(super::UnableToGetReadProviderSnafu {
                     dataconnector: "debezium",
+                    connector_component: ConnectorComponent::from(dataset),
                 })?;
 
                 ensure!(
                     topic == metadata.topic,
                     super::InvalidConfigurationNoSourceSnafu {
                         dataconnector: "debezium",
-                        message: format!("The topic has changed from {} to {topic} for dataset {dataset_name}. The existing accelerator data may be out of date.", metadata.topic),
+                        message: format!("The topic has changed from {} to {topic}. The existing accelerator data may be out of date.", metadata.topic), // TODO: what action can a user take from this error?
```

Comment:
I will address this `TODO:` in a future PR as part of #3506

Reply:
For this error, the user would need to delete the acceleration DB file. We could probably add this to the docs and link to that section here.